### ISSUE TITLE
MySQL 8.0.2+ implementations throw syntax errors due to field/keyword conflicts

### ISSUE THREAD

**Issue by ivanbiljan** [Pryaxis/TShock#1675]
See screenshot below:
![image](https://user-images.githubusercontent.com/7541476/61496271-f2438300-a9bb-11e9-93cc-85a23eee6bd0.png)

As per @ASgoPew, similar erros are thrown when defining regions.

**Comment by QuiCM** [comment]
Is this due to a bee MySQL version? I don't recall this issue existing in the past.
Regardless, we should look at the table creation code and see if we can drop some escaping in

**Comment by ivanbiljan** [comment]
Yes, `Groups` was added in 8.0.2. Reference: https://dev.mysql.com/doc/refman/8.0/en/keywords.html#keywords-8-0-detailed-G
